### ISSUE TITLE
NVDA repeating or skipping while navigating or while reading all in some web pages

### ISSUE THREAD

**Issue by Surveyor123** [nvaccess/nvda#10193]
### Steps to reproduce:
Open for instance this page:
https://www.medicalpark.com.tr/menenjit-nedir-belirtileri-ve-tedavi-yontemleri-nelerdir/hg-1863
Press 1 on the number row for moving text in heading 1
Give sayAll command
### Actual behavior:
NVDA skips or read some words twice while reading.
### Expected behavior:
NVDA should read as it should be
#### NVDA installed/portable/running from source:
It's valid for both installed and portable coppies
#### NVDA version:
it's valid for 2019.2 and 2019.3 alphas
#### Windows version:
Windows 10Ver1903 (64-bit) build 18362.329
#### Name and version of other software in use when reproducing the issue:
it can be reproduced using Chrome or Firefox
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, the same

Some peace of log:
accRole failed: (-2147024809, 'Parametre hatali.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (14:06:52.474):
accRole failed: (-2147024809, 'Parametre hatali.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (14:06:52.655):
accRole failed: (-2147024809, 'Parametre hatali.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (14:06:52.836):
accRole failed: (-2147024809, 'Parametre hatali.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (14:06:53.016):
accRole failed: (-2147024809, 'Parametre hatali.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (14:06:53.186):
accRole failed: (-2147024809, 'Parametre hatali.', (None, None, None, 0, None))

**Comment by Adriani90** [comment]
cc: @leonardder

**Comment by zahra21** [comment]
hi.
can you please send a website in english?
i have nvda since 2014 and i dont remember such behaviour since 2014.
i did not work with newer versions of nvda or different browsers which dont support windows xp.

**Comment by Surveyor123** [comment]
Hi,
Unfortunately I don’t have a site in English for wich you can reproduce this.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by Adriani90** [comment]
@Surveyor123 how is this now in NVDA 2020.1?

**Comment by Surveyor123** [comment]
Unfortunately nothing has changed.

**Comment by Surveyor123** [comment]
I'm Testing NVDA 2021.1 and the same issue still persists.
Here is another site on which it can be reproduced:
https://www.hukukihaber.net/belgeye-dayanmayan-icra-takibine-karsi-acilan-menfi-tespit-ve-istirdat-davalari-makale,9063.html

**Comment by Adriani90** [comment]
@Surveyor123 are you still able to reproduce this iwth NvDA 2023.1 RC1? Please make sure that automatically move system focus to focusable elements is turned off in NVDA browse mode settings.

**Comment by Surveyor123** [comment]
@Adriani90  Unfortunately switching the automatically move system focus to focusable elements on or off makes no difference for this issue.

**Comment by Adriani90** [comment]
Ok this seems weird. I think there is something in the javascript code that scrolls the page automatically or something like this. But I find it really hard to investigate where it comes from. It might be related to#8010. I guess we need a minimal test case here.

**Comment by Surveyor123** [comment]
@Adriani90 I'd like to add that the issue is not reproduced with JAWS or Narrator. And NVDA narrates propperly if reloaded after the page in question loads.

**Comment by Adriani90** [comment]
Closing this in favor of #15338 which contains some more techical details.
